test/dynamo/test_package.py

```diff
@@ -0,0 +1,196 @@
+# Owner(s): ["module: dynamo"]
+
+import os
+import pickle
+
+import torch
+import torch._dynamo.testing
+import torch._inductor.config
+import torch._inductor.test_case
+import torch.onnx.operators
+import torch.utils.cpp_extension
+from torch._dynamo.package import CompilePackage
+from torch._inductor.runtime.runtime_utils import cache_dir
+
+
+class StorageForTesting:
```

Comment:
Is this implementing an interface that's going to be in FB? Would be nice to have a protocol for it.